runner/ansible/roles/2.1.4/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.4'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # if virtIP not configured, exit with fail
+    cibadmin -Q --xpath "//primitive[@type='IPaddr2']/@type" || exit 1
+    # if not grouped, exit with fail
+    cibadmin -Q --xpath "//group/primitive[@type='IPaddr2']/@type" || exit 1
```

Comment:
This grouped check might be risky, as it only belongs to azure. Could we check here if the cloud provider is azure to run this check (only the `//group/primitive[@type='IPaddr2']/@type` one)

Reply:
I am cool with making that change, but as with the others, I can't do it at this time.